tools/db_import/db_import/rules.py

```diff
@@ -0,0 +1,100 @@
+## Copyright 2023 The OpenXLA Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import csv, datetime, io, json, re, os
+from typing import Any, Optional, Callable
+import rjsonnet
+from google.cloud import storage
+from db_import.utils import first_no_except
```

Comment:
Could you sort imports (the first part, e.g., `from *`, `import *`) in lexicographical order?